testsuite/grpc-tests/src/main/java/dev/resteasy/grpc/lists/sets/D1.java

```diff
@@ -0,0 +1,75 @@
+package dev.resteasy.grpc.lists.sets;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class D1<T> {
+
+    ArrayList<T> at = new ArrayList<T>();
+    ArrayList<Integer> l00 = new ArrayList<Integer>();;
+    ArrayList<?> l01 = new ArrayList<Byte>();
+    ArrayList<S1> l02 = new ArrayList<S1>();
+    ArrayList<S2> l03 = new ArrayList<S2>();
+    ArrayList<S3<String>> l04 = new ArrayList<S3<String>>();
+
+    ArrayList<List<Float>> l10 = new ArrayList<List<Float>>();
+    ArrayList<List<?>> l11 = new ArrayList<List<?>>();
+    ArrayList<List<S1>> l12 = new ArrayList<List<S1>>();
+    ArrayList<List<S2>> l13 = new ArrayList<List<S2>>();
+    ArrayList<List<S3<Integer>>> l14 = new ArrayList<List<S3<Integer>>>();
+
+    List<Integer> l20 = new ArrayList<Integer>();
+    List<?> l21 = new ArrayList<Short>();
+    List<S1> l22 = new ArrayList<S1>();
+    List<S2> l23 = new ArrayList<S2>();
+    List<S3<Float>> l24 = new ArrayList<S3<Float>>();
+
+    List<List<Float>> l30 = new ArrayList<List<Float>>();
+    List<List<?>> l31 = new ArrayList<List<?>>();
+    List<List<S1>> l32 = new ArrayList<List<S1>>();
+    List<List<S2>> l33 = new ArrayList<List<S2>>();
+    List<List<S3<Long>>> l34 = new ArrayList<List<S3<Long>>>();
+
+    List<ArrayList<Float>> l40 = new ArrayList<ArrayList<Float>>();
+    List<ArrayList<?>> l41 = new ArrayList<ArrayList<?>>();
+    List<ArrayList<S1>> l42 = new ArrayList<ArrayList<S1>>();
+    List<ArrayList<S2>> l43 = new ArrayList<ArrayList<S2>>();
+    List<ArrayList<S3<Long>>> l44 = new ArrayList<ArrayList<S3<Long>>>();
+
+    List l50 = new ArrayList();
+    ArrayList l51 = new ArrayList();
+
+    public boolean equals(Object o) {
+        if (!D1.class.equals(o.getClass())) {
+            return false;
+        }
```

Comment:
I assume this is only used for testing. However, there is a potential NPE here. `o` could be `null`.

This seems to be the same in some of the ones below as well. I won't comment on each one :)

Reply:
Fixed them all.